src/core/ugbio_core/logger.py

```diff
@@ -1,8 +1,16 @@
 import logging
+import sys
 
 # create logger
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
 # create formatter
 formatter = logging.Formatter("%(asctime)s - %(module)s - %(levelname)s - %(message)s")
+
+# create console handler and set level to debug
```

Comment:
Is the level not INFO?

Reply:
you're right. I copy pasted it from VariantCalling without read the comment. thanks!